source/_patterns/basics/forms/input.hbs

```diff
@@ -3,3 +3,72 @@
   <input type="text" class="form-control" id="inlineFormInput" placeholder="Jane Doe">
   <button type="submit" class="btn btn-primary">Submit</button>
 </form>
+
+<form>
```

Comment:
With the addition of all this, the Component-level Search form is off, because it's including all of this. I think each form group should be its own basic form pattern so that they can be included elsewhere easily.